js/packages/go-bridge/defaults.ts

```diff
@@ -1,4 +1,9 @@
-export const GoBridgeDefaultOpts = {
+export type GoBridgeOpts = {
+	cliArgs: string[]
+	persistence: boolean
+}
```

Comment:
```suggestion
export type GoBridgeOpts = {
	cliArgs: string[]
	persistence: boolean
}
import { GoBridgeOpts } from "./types"
```

Reply:
Oh i don't see it ... sorry ... and thx 👍